### PR TITLE
Possible to use present and current for to in period

### PR DESCRIPTION
**Pull request by megamanxl**

### CODE REVIEW

**Comment by gregoryv** [comment]
I like the idea; but you don't have to switch the types for this. Instead of allowing textual "present" and "current" values, let's just document it as if a year is '0' or maybe '-1' it means time.Now().Year().
That way we 
- add the feature of using "present" year and
- keep the benefit of having the values as integer which simplifies some calculations

**Comment by gregoryv** [comment]
If we say 0 means time.Now().Year() you can actually just leave the field empty in you yaml file, as that is the default value.
Of course we won't be able to document when we worked from 0 BC ;-) but I guess we can live with that.

File: templates.go [hunk 1]
```diff
@@ -167,19 +168,28 @@ func NewCVPage(co *Company, in *CV) *Page {
...
-		if p.Period.ToYear > to {
-			to = p.Period.ToYear
+		if ToYear, err := strconv.Atoi(p.Period.ToYear); err == nil {
+			if ToYear > to {
+				to = ToYear
+			}
+		} else if p.Period.ToYear == "present" || p.Period.ToYear == "current" {
+			to = time.Now().Year()
 		}
 	}
```
Comment (gregoryv) [inline templates.go]: I guess you would only really need to add

```suggestion
if to == 0 {
	to = time.Now().Year()
}
```

Comment (megamanxl) [reply templates.go]: I'll check, and yes it is a valid solution.

**Comment by gregoryv** [comment]
Created issue #4 for this.

**Pull request merged**